_data/form.yml

```diff
@@ -0,0 +1,831 @@
+
+      After receiving an Owner-Builder permit, you may decide to hire Contractors instead. If this happens, you will need to verify whether they:
+
+      - Are licensed in California
+
+      - Have current workers' compensation insurance coverage
+
+  - type: header
+    level: 5
+    label: Resources
+
+  - type: paragraph
+    label: |
+      To learn more about your obligations as an "employer," you can contact the:
+
+      - Internal Revenue Service
+      - United States Small Business Administration
+      - California Department of Benefit Payments
+      - California Division of Industrial Accidents.
+
+      To learn more about licensed contractors, contact the California Contractors’ State License Board.
+
+      - **Phone:** 1-800-321-CSLB (2752)
+
+      - **Website:** [www.cslb.ca.gov](www.cslb.ca.gov)
+
+  - type: checkbox
+    label: I understand and agree to everything on this page.
+
+- title: Disclosures for Owner-Builders
+  group: owner-builder
+  fields:
+  - type: header
+    level: 5
+    label: Information for Owner-Builders
+
+  - type: paragraph
+    label: |
+
+      If you sell the property listed on this permit, and there are latent defects in the workmanship or materials, you may be held liable for the financial or personal injuries any future owner(s) sustain.
+
+      Also, under California Contractors' State License Law, it is illegal to build a single-family residential structure with the intent to sell it, unless you:
+
+      - Contract a licensed general building Contractor for all work
+
+      - Hire licensed subcontractors for 4 structures or less within a calendar year
+
+  - type: checkbox
+    label: I understand and agree to everything on this page.
+
+- title: Disclosures for Owner-Builders
+  group: owner-builder
+  fields:
+  - type: header
+    level: 5
+    label: "By checking the box below:"
+
+  - type: paragraph
+    label: |
+
+      You declare that you&hellip;
+
+      - Consent to and are aware of the Owner-Builder building permit applied for in your name
+
+      - Are willfully acting as an Owner-Builder
+
+      - Understand the limits of your insurance coverage for injuries to workers on your property
+
+      - Understand you are legally and financially responsible for the proposed work at the project address
+
+      You also agree to&hellip;
+
+      - Follow all applicable laws and requirements for Owner-Builders and employers
+
+      - Notify the city immediately if you need to add, remove, or change any information on this form
+
+  - type: checkbox
+    label: I understand and agree to everything on this page.
+
+- title: Almost done!
+  description: All that's left is to find out whether you will need to submit more forms.
+  fields:
+  - label: Will the work involve any of the following?
```

Comment:
This could be a static content page for v1

Reply:
Look at cannabis forms (business type) for an example of this "back-and-forth" between agencies and applicants